examples/python/CloudantDF.py

```diff
@@ -45,7 +45,7 @@
 
 df.filter(df.airportName >= 'Moscow').select("_id",'airportName').show()
 df.filter(df._id >= 'CAA').select("_id",'airportName').show()
-df.filter(df._id >= 'CAA').select("_id",'airportName').save("airportcodemapping_df",
+df.filter(df._id >= 'CAA').select("_id",'airportName').write.save("airportcodemapping_df",
         "com.cloudant.spark", bulkSize = "100")
```

Comment:
@HolgerKache Holger, this patch was intended when number of partitions > number of documents to be saved. In your case, the database is very small, it uses just one partition, so save function works fine.
Example to demonstrate the work of the patch:

```
conf.set("jsonstore.rdd.partitions",20)  #using 20 partitions
df = sqlContext.load("n_flight", "com.cloudant.spark") # big enough database to use all partitions
df2 = df.filter(df.flightSegmentId=='AA106').select("*") #df2 contains only 5 docs
df2.write.save("n_flight111",  "com.cloudant.spark", 
     bulkSize = "100", createDBOnSave="true")  #this will throw an error without the patch

```